scripts/zones/Mount_Zhayolm/mobs/Cerberus.lua

```diff
@@ -37,5 +37,5 @@ end;
 -----------------------------------
 
 function onMobDespawn(mob)
-    mob:setRespawnTime((math.random(0,24)*3600)+172800); -- 48-72 hours proper 1 hour windows
+    mob:setRespawnTime(math.random(172800,259200)); -- 48-72 hours proper 1 hour windows
```

Comment:
lol, easier than trying to rewrite every other one to keep the same consistency xD
Also easier for my master plan, to take over the world!
...I mean, to create a separate timed spawn lua...